keras_cv/layers/object_detection/ssd_head.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2023 The KerasCV Authors
```

Comment:
you may need to create separate directory for ssd. see others https://github.com/keras-team/keras-cv/tree/master/keras_cv/models/object_detection

Reply:
Fixed. Can you please review PR ?